src/main/java/use_cases/PostManager.java

```diff
@@ -53,10 +53,10 @@ private boolean interactPost(Post targetPost, Object obj, boolean type){
 
     /**
      * Creates a post and adds it to the list of post
-     * @param owner
-     * @param postedTime
-     * @param recipe
-     * @param category
+     * @param owner: who created the post
```

Comment:
Sorry to nitpick, but could we please remove the colons on these param comments for style consistency (i.e. "owner The User who created the Post")? This webpage from Oracle has examples if you want to take a look https://www.oracle.com/ca-en/technical-resources/articles/java/javadoc-tool.html